src/routes/docs/advanced/security/mfa/+page.markdoc

```diff
@@ -0,0 +1,36 @@
+---
+layout: article
+title: Multi-factor Authentication
+description: Appwrite helps you secure your developer accounts with MFA (multi-factor authentication).
```

Comment:
Not just `developer accounts`. Thats probably true for use in Console, but priority is that you can use it when building your own app for your end users